File: resources/expected/ros2/fan_in_fan_out_inverted_topics/lax/src/fan_in_fan_out_system_cpp_pkg/src/base_code/fanIn_producer1_base_src.cpp
#include "fan_in_fan_out_system_cpp_pkg/base_headers/fanIn_producer1_base_src.hpp"

//========================================================
// Re-running Codegen will overwrite changes to this file
//========================================================

fanIn_producer1_base::fanIn_producer1_base() : Node("fanIn_producer1")
{
    cb_group_ = this->create_callback_group(rclcpp::CallbackGroupType::Reentrant);
    // Setting up connections
    fanIn_producer1_myInteger_publisher_ = this->create_publisher<fan_in_fan_out_system_cpp_pkg_interfaces::msg::Integer64>(
        "fanIn_producer1_myInteger",
        1);

    // timeTriggered callback timer
    periodTimer_ = this->create_wall_timer(std::chrono::milliseconds(1000),
        std::bind(&fanIn_producer1_base::timeTriggered, this), cb_group_);

}

//=================================================
//  C o m m u n i c a t i o n
//=================================================

void fanIn_producer1_base::put_myInteger(fan_in_fan_out_system_cpp_pkg_interfaces::msg::Integer64 msg)
{
    fanIn_producer1_myInteger_publisher_->publish(msg);
}

File: resources/expected/ros2/python-fan_in_fan_out_inverted_topics/lax/src/fan_in_fan_out_system_py_pkg/fan_in_fan_out_system_py_pkg/base_code/fanIn_producer1_base_src.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from queue import Queue
from rclpy.callback_groups import ReentrantCallbackGroup
from fan_in_fan_out_system_py_pkg_interfaces.msg import Integer64

#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================

class fanIn_producer1_base(Node):
    def __init__(self):
        super().__init__("fanIn_producer1")

        self.cb_group_ = ReentrantCallbackGroup()

        # Setting up connections
        self.fanIn_producer1_myInteger_publisher_ = self.create_publisher(
            Integer64,
            "fanIn_producer1_myInteger",
            1)

        # timeTriggered callback timer
        self.periodTimer_ = self.create_timer(1000, self.timeTriggered, callback_group=self.cb_group_)

    def timeTriggered(self):
        pass

    #=================================================
    #  C o m m u n i c a t i o n
    #=================================================

    def put_myInteger(self, msg):
        typedMsg = Integer64()
        typedMsg.data = msg
        self.fanIn_producer1_myInteger_publisher_.publish(typedMsg)

